core/client/ui/onboarding.js

```diff
@@ -0,0 +1,32 @@
+Template.onboarding.onCreated(async function () {
+    const constraints = userStreams.getStreamConstraints(streamTypes.main);
+    const stream = await userStreams.requestUserMedia(constraints);
+    if (!stream) { lp.notif.error(`unable to get a valid stream`); return; }
+
+    if (Session.get("sceneWorldReady")) {
+        // We disable keyboard to not let the player move while onboarding
+        const worldScene = game.scene.getScene('WorldScene');
+        worldScene.enableKeyboard(false)
+    }
+
+    Session.set('streamAccepted', true)
+
+    // We should stop the stream directly after asking permissions, since we just want to check if the user has granted permissions
+    userStreams.destroyStream(streamTypes.main);
+});
+
+Template.onboarding.events({
+    'click .continue-button'() {
+        Meteor.users.update(Meteor.userId(), { $unset: { 'profile.guest': true } });
+        
+        const worldScene = game.scene.getScene('WorldScene');
+        worldScene.enableKeyboard(true)
```

Comment:
Missing semi-colon, you should enable auto-formatting to avoid bothering with this ;)